ls1intum/Artemis
# Fix sentry issue for guided-tour.service

### Motivation and Context



This PR fixes `ARTEMIS-4` sentry issue. `currentTourStep` or `currentTourStep.highlightSelector` are possibly undefined. added simple checks for that. 
 
### Steps for Testing


To able start/restart/continue Guided Tutorial,  the user has to be part of Tutor group of TutorialCourse in test server. (Find `Tutorial Course (artemistutorial)`, then in course detail view, add your user into `Tutors`)

When the user is not part of that, following these steps sometimes causes the errors respectively. If your user is part of tutor group of TutorialCourse, guided tutorial works as expected.

1. Log in to Artemis as tutor or higher user role
2. Navigate to Course Management
3. Select `Start/Restart tutorial` from menu, and try other routes like go to Course page or go back CoruseManagement. The following errors should not be displayed at the console.

`Cannot read property 'highlightSelector' of undefined`
`Cannot read property 'orientation' of undefined`
`Cannot read property 'closeAction' of undefined`
